src/PhpSpec/ObjectBehavior.php

```diff
@@ -32,6 +32,9 @@
  * @method void beConstructedThrough($factoryMethod, array $constructorArguments)
  * @method void beAnInstanceOf($class)
  * @method void shouldHaveType($type)
+ * @method void shouldImplement($class)
+ * @method void shouldBeAnInstanceOf($class)
+ * @method void shouldReturnAnInstanceOf($class)
```

Comment:
The method exists but in the case of calling it on the object being tested, I don't think it's something to encourage.

If you remove that line I'll merge the others.